modules/adding-a-model-server-to-your-data-science-project.adoc

```diff
@@ -17,7 +17,7 @@ endif::[]
 * You have created a data science project that you can add a model server to.
 ifndef::upstream[]
 * If you want to use a custom model-serving runtime for your model server, you have added and enabled the runtime. See link:{rhodsdocshome}{default-format-url}/working_on_data_science_projects/working-on-data-science-projects_nb-server#adding-a-custom-model-serving-runtime_nb-server[Adding a custom model-serving runtime].
-* If you want to use graphics processing units (GPUs) with your model server, you have enabled GPU support in {productname-short}. See {rhodsdocshome}{default-format-url}/managing_users_and_user_resources/enabling-gpu-support-in-openshift-data-science_user-mgmt[Enabling GPU support in {productname-short}]
+* If you want to use graphics processing units (GPUs) with your model server, you have enabled GPU support in {productname-short}. See {rhodsdocshome}{default-format-url}/managing_resources#enabling-gpu-support-in-openshift-data-science[Enabling GPU support in {productname-short}]
```

Comment:
Should we conditionalize this link for upstream/downstream?

Reply:
yes - fixed